Classes/Service/Reporting.php

```diff
@@ -104,7 +104,13 @@ public function getNodeStat(NodeInterface $node, ControllerContext $controllerCo
 
         $nodeUri = $this->getLiveNodeUri($node, $controllerContext);
         $hostname = !empty($siteConfiguration['overrideHostname']) ? $siteConfiguration['overrideHostname'] : $nodeUri->getHost();
-        $filters = 'ga:pagePath==' . $nodeUri->getPath() . ';ga:hostname==' . $hostname;
+        $pagePathPrefix = !empty($siteConfiguration['pagePathPrefix']) ? $siteConfiguration['pagePathPrefix'] : '';
```

Comment:
```suggestion
        $pagePathPrefix = $siteConfiguration['pagePathPrefix'] ?? '';
```

Reply:
Does the empty check even work when the value is not set at all?

Reply:
Yes, it does 😄 We have this one running as a patch